src/inmanta/agent/handler.py

```diff
@@ -221,6 +231,20 @@ def _log_msg(
         raise NotImplementedError
 
 
+class ContextResourceState(str, Enum):
+    """
+    Used by the HandlerContext to determine the state of the resource
+    Similar to const.ResourceState but with some states removed and with the added skipped_for_dependency state
```

Comment:
Yes, that is my understanding of it. Thank you for pointing out this, I sometimes struggle to write proper docstrings. I'll start with the `is` instead of the `how` in the future